htdocs/themes/math4/math4.css

```
@@ -323,6 +323,10 @@ h2.page-title {
     text-align:center;
 }
 
+.new-course-options label span {
+	vertical-align: middle;
+}
+
```

This doesn't work as expected.  I assume you are trying to make the label and checkbox align better.   For Google Chrome it improves this a bit, but for Firefox it doesn't change a thing.  The best that I have found for this in this case that is good for both of these browsers is:
```suggestion
.new-course-options label.checkbox input {
	margin-top: 0;
	float: unset;
	vertical-align: unset;
}
```

I can't guarantee this be an improvement on other browsers though.

The real problem here is bootstrap 2.3.2.  It is adding style that doesn't work right, and causes the misalignment.  That style may have worked 5 years ago, but browsers have changed.  The style changes I put above are undoing what bootstrap does in this instance.  The proper set up for browsers now is to not have the input inside its own label.  Newer versions of bootstrap are updated for that.  Although, to get that to work correctly you also have to add their form-control styles.

I am not sure that it is worth it to fix the appearance of these checkboxes and bootstrap 2.3.2's problems.  The other labels and inputs on the page are badly misaligned as well, so this is a very minor improvement.

---

Agreed that this is not worth too much effort/investigation. It's just for a user of the admin page to see.

If you confirm that your suggestion is indeed better than the `vertical-align`, I'll accept that change here. But as I noted, for me on a Mac with Firefox, the `vertical-align` makes it work.